xarray/tests/test_groupby.py

```diff
@@ -663,30 +663,33 @@ def test_groupby_dataset_reduce() -> None:
     assert_allclose(expected, actual)
 
 
-def test_groupby_dataset_math() -> None:
+@pytest.mark.parametrize("squeeze", [True, False])
+def test_groupby_dataset_math(squeeze) -> None:
```

Comment:
`squeeze` does not seem to make a difference for `expected` - is that to be expected or do I miss something?

Reply:
I think it's right. 

Squeeze controls whether `dim1` is squeezed out when iterating over groups. It doesn't matter here because we end up concatenating  along `dim1` anyway at the end so it comes back.